runtime/gc_glue_java/JNICriticalRegion.cpp

```diff
@@ -36,6 +36,10 @@
 void
 MM_JNICriticalRegion::reacquireAccess(J9VMThread* vmThread, UDATA accessMask)
 {
+#if defined(J9VM_INTERP_ATOMIC_FREE_JNI)
+	// Current thread must have entered the VM before acquiring exclusive
```

Comment:
I am wondering if these asserts could be enabled all of the time? Not asking you to change them just thinking out loud.

Reply:
Sorry thought that was a VMAccess check lol.  My bad!